### ISSUE TITLE
Could you share the three data files?

### ISSUE THREAD

**Issue by Sunscreen123** [irskid5/drr_restaurants#2]
Dear Vele Tosevski: 
       After reading your code, the idea is very clear and easy to understand. Could you share the three data files：
RESTAURANTS_PATH = 'dataset/processed_rest.csv'
REVIEWS_PATH = 'dataset/reviews.csv'
USERS_PATH = 'dataset/processed_users.csv'

**Comment by Fewrenge** [comment]
@Sunscreen123 
Hi there, I'm having the same problem here. I assume that there need to be some other data processing procedures since I can't find any Yelp dataset that comes in CSV format. Would you mind telling me how did you solve this problem? Any help would be deeply appreciated.

**Comment by Fewrenge** [comment]
Update: I found the official JSON to CSV converter from Yelp's dataset examples pages on GitHub (links below) after I wrote my own converter that also does the job of dropping useless attributes. As long as the CSV data files that you feed into the preprocessing procedure are correct, the entire code is good to go. I only modified parts of the program that involve output and paths and fixed some bugs that I encountered when running the program on Colab. Mainly just “tensors used as indices must be long, byte or bool tensors” and could be easily fixed by changing the tensor type of the line that reports the error in the corresponding python files. Not sure if they're mere bugs or errors due to version differences since I'm not very experienced in Pytorch.
Link: https://github.com/Yelp/dataset-examples/blob/master/json_to_csv_converter.py
